regulations/static/components/RelatedRules.vue

```diff
@@ -22,25 +25,41 @@ export default {
         },
         limit: {
             type: Number,
-            default: 3,
+            default: 5,
         },
     },
 
     data() {
         return {
-            rules: null,
+            rules: [],
+            limitedList: true,
         }
     },
 
+    computed: {
+      limitedRules() {
+        if(this.limitedList) {
+          return this.rules.slice(0, this.limit);
+        }
+        return this.rules;
+      },
+      rulesCount() {
+        return this.rules.length;
+      }
+    },
+
     async created() {
-        this.rules = await this.fetch_rules(this.title, this.part, this.limit);
+        this.rules = await this.fetch_rules(this.title, this.part);
     },
 
     methods: {
-        async fetch_rules(title, part, limit) {
-            const response = await fetch(`https://www.federalregister.gov/api/v1/documents.json?fields[]=type&fields[]=abstract&fields[]=citation&fields[]=correction_of&fields[]=dates&fields[]=docket_id&fields[]=docket_ids&fields[]=document_number&fields[]=effective_on&fields[]=html_url&fields[]=publication_date&fields[]=regulation_id_number_info&fields[]=regulation_id_numbers&fields[]=title&order=newest&conditions[type][]=RULE&conditions[cfr][title]=${title}&conditions[cfr][part]=${part}&per_page=${limit}`);
+        async fetch_rules(title, part) {
```

Comment:
This defaults to 20, which I think is a reasonable outside limit to set, and something that can be tweaked.